src/App.js

```diff
@@ -12,30 +12,36 @@ import TopicsList from './components/TopicsList';
 import TopicsPage from './components/TopicsPage';
 import CommentsList from './components/CommentsList';
 import CommentsAdder from './components/CommentsAdder';
+import { UserContext } from './contexts/User';
+import { useContext } from 'react'
+import User from './components/User';
 
 function App() {
   const [articlesLists, setArticlesLists] = useState([]);
+  const value = useContext(UserContext);
```

Comment:
it might be better to use a different name than 'value' to show what it is.  a logged in user? username?